test/e2e/e2e.test.js

```diff
@@ -1,45 +1,16 @@
-const { APP_NAME } = require('../../src/constants');
+
 const DeleteCommand = require('../../src/commands/rtc/apps/video/delete');
 const DeployCommand = require('../../src/commands/rtc/apps/video/deploy');
 const ViewCommand = require('../../src/commands/rtc/apps/video/view');
 
-const jwt = require('jsonwebtoken');
-const { nanoid } = require('nanoid');
-const path = require('path');
 const { stdout } = require('stdout-stderr');
-const superagent = require('superagent');
 
-const twilioClient = require('twilio')(process.env.TWILIO_API_KEY, process.env.TWILIO_API_SECRET, {
```

Comment:
@shrutiburman are these account credentials configured by the customer when they deploy the React / iOS / Android app? Or are you saying these are associated with a Twilio account?

Reply:
These account creds are configured on circle CI env and are referenced from there. These are associated with a Twilio account.